examples/verifying_fd_afd.py

```diff
@@ -0,0 +1,93 @@
+import desbordante
+import pandas as pd
+
+
+GREEN_CODE = '\033[1;42m'
+RED_CODE = '\033[1;41m'
+BLUE_CODE = '\033[1;46m'
+DEFAULT_COLOR_CODE = '\033[1;49m'
+DATA_FOR_FD = pd.read_csv('examples/datasets/duplicates_short.csv', header=[0])
+DATA_FOR_AFD = pd.read_csv('examples/datasets/DnD.csv', header=[0])
+
+
+def print_clusters(verifier, data, lhs, rhs):
+    print(f'Number of clusters violating FD: {verifier.get_num_error_clusters()}')
+    for i, highlight in enumerate(verifier.get_highlights()):
+        print(f"{BLUE_CODE} #{i+1} cluster: {DEFAULT_COLOR_CODE}")
+        for el in highlight.cluster:
+            print(f"{el}: {data[data.columns[lhs]][el]} -> {data[data.columns[rhs]][el]}")
+        
+        print(f"Most frequent rhs value proportion: {highlight.most_frequent_rhs_value_proportion}")
+        print(f"Num distinct rhs values: {highlight.num_distinct_rhs_values}\n")
+
+
+def print_results_for_fd(verifier, lhs, rhs):
+    if verifier.fd_holds():
+        print(GREEN_CODE, 'FD holds', DEFAULT_COLOR_CODE)
+    else:
+        print(RED_CODE, 'FD does not hold', DEFAULT_COLOR_CODE)
+        print_clusters(verifier, DATA_FOR_FD, lhs, rhs)
+        # print(f"But the same {GREEN_CODE} AFD with error threshold = {verifier.get_error()} holds{DEFAULT_COLOR_CODE}")
```

Comment:
Commented code

Reply:
We decided to leave it as this since this invocation clutters the output. However, it is useful to show that clusters can be obtained while using the AFD primitive too.